src/ui/OpenchannelOGMessage/index.tsx

```diff
@@ -61,27 +61,32 @@ export default function OpenchannelOGMessage({
   userId,
 }: OpenChannelOGMessageProps): JSX.Element {
   if (!message || message.messageType !== 'user') {
-    return null;
+    return <></>;
```

Comment:
This return here is not safe. it would possibly break rule of hooks(very improbable)
, better move to it to line 113 (right before the final return)

Reply:
!important ^^